Scenes/MainMenu/MainMenu.gd

```diff
@@ -0,0 +1,60 @@
+extends Node2D
+
+onready var WEBSITE_PATH = "http://elf-vs-dwarves.pl/"
+onready var SAVE_FILE_PATH = "save.json"
+
+onready var world_path = "res://Scenes/World/World.tscn"
+
+onready var home = get_node("Canvas/Home")
+onready var options = get_node("Canvas/Options")
+onready var about = get_node("Canvas/About")
+
+
+
+func _on_ContinueBtn_pressed():
+	get_tree().change_scene(world_path)
+
+
+func _on_NewGameBtn_pressed():
+	Directory.new().remove(SAVE_FILE_PATH)
+	get_tree().change_scene(world_path)
+
+
+func _on_OptionsBtn_pressed():
+	switch_panel(MENU_PANEL.OPTIONS)
+
+
+func _on_AboutBtn_pressed():
+	switch_panel(MENU_PANEL.ABOUT)
+
+
+func _on_ExitBtn_pressed():
+	get_tree().quit()
+
+func switch_panel(var menu_panel):
+	match menu_panel:
+		MENU_PANEL.HOME:
+			options.visible = false
+			home.visible = true
+			about.visible = false
+		MENU_PANEL.OPTIONS:
+			home.visible = false
+			options.visible = true
+			about.visible = false
+		MENU_PANEL.ABOUT:
+			home.visible = false
+			options.visible = false
+			about.visible = true
```

Comment:
Maybe first make invisible every panels, and thet make visible one panel in `match` statement? It will be less code. But maybe we don't have to set visibility of other panels to false, because those panels are invisible at start? Only set home visibility to false and selected panel visibility to true